packages/client/mutations/UpdateJiraDimensionFieldMutation.ts

```diff
@@ -99,19 +100,22 @@ const UpdateJiraDimensionFieldMutation: SimpleMutation<TUpdateJiraDimensionField
       // handle meeting records
       const phases = meeting.getLinkedRecords('phases')
       const estimatePhase = phases.find((phase) => phase.getValue('phaseType') === 'ESTIMATE')!
-      const stages = estimatePhase.getLinkedRecords('stages')
+      const stages = estimatePhase.getLinkedRecords<JiraFieldMenu_stage[]>('stages')
       stages.forEach((stage) => {
-        const {cloudId: stageCloudId} = JiraIssueId.split(
-          stage.getValue('serviceTaskId') as string
-        )
-        if (stage.getValue('dimensionName') === dimensionName && stageCloudId === cloudId) {
+        const dimensionRef = stage.getLinkedRecord('dimensionRef')
+        const dimensionRefName = dimensionRef.getValue('name')
+        if (dimensionRefName !== dimensionName) return
+        const task = stage.getLinkedRecord('task')
+        const _integration = task.getLinkedRecord('integration')
+        if (_integration.getValue('__typename') !== 'JiraIssue') return
+        const integration = _integration as DiscriminateProxy<typeof _integration, 'JiraIssue'>
```

Comment:
this just tells typescript that the integration is a proxy of  a JiraIssue (vs. GitHub or anything else).
My PR to relay is pending, so this is how we have to do it for now